Atomic/syscontainers.py

```diff
@@ -106,7 +106,12 @@ def _checkout_layer(self, repo, rootfs_fd, rootfs, rev):
                             stdin=DEVNULL,
                             stdout=DEVNULL,
                             stderr=DEVNULL)
-
+            if self.user:
+                for root,dirs,_ in os.walk(rootfs):
+                    for d in dirs:
+                        dirname = os.path.join(root,d)
+                        os.chmod(dirname,os.stat(dirname).st_mode | stat.S_IWUSR)
+                        
```

Comment:
this should not be necessary now, as it is already done on importing the layers.  Could you verify that it works without this code?